test/setup/index.js

```diff
@@ -14,7 +14,7 @@ export function createTestContext(opts = {}) {
 }
 
 export const fixture = {
-  gl: createTestContext(),
+  gl: createTestContext({webgl2: false, webgl1: true}),
```

Comment:
createTestContext() with no input options using default options and hence creating WebGL2 context, I made this change to create WebGL1 context.